CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## 1.25.0 (Unreleased)
+> **_NOTE:_** We are moved to `releasenotes`. The latest CHANGELOG could be found [here](https://docs.otc-service.com/releasenotes/terraform-provider-opentelekomcloud/).
```

Comment:
If "latest" is mentioned, link should be to the https://docs.otc-service.com/releasenotes/terraform-provider-opentelekomcloud/current.html